File: 1899-count-items-matching-a-rule/1899-count-items-matching-a-rule.java
class Solution {
    public int countMatches(List<List<String>> items, String key, String val) {
        int j;
        int count=0;
        if(key.equals("type"))
        {
            j=0;
        }
        else if(key.equals("color"))
        {
            j=1;
        }
        else
        {
            j=2;
        }
        for(List<String> i : items)
        {
            if(i.get(j).equals(val))
            {
                count++;
            }
        }
        return count;
        
    }
}

File: 1899-count-items-matching-a-rule/1899-count-items-matching-a-rule.py
class Solution:
    def countMatches(self, items: List[List[str]], key: str, val: str) -> int:
        count = 0
        if key =="type":
            j=0
        elif key =="color":
            j=1
        else:
            j=2
        for i in items:
            if val == i[j]:
                count+=1
        return count
        